src/main/java/mate/controller/IndexController.java

```diff
@@ -2,14 +2,18 @@
 
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+@WebServlet(urlPatterns = "/")
 public class IndexController extends HttpServlet {
+    private static final String PATH = "/WEB-INF/views/index.jsp";
+
     @Override
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+        req.getRequestDispatcher(PATH).forward(req, resp);
```

Comment:
actually it would be better to use string literals instead of constants
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
```